File: src/test/resources/expectedfolder/ResponseContent.cs
using System;

namespace com.github.ggerla.java2csharp.testclasses
{
	public class ResponseContent
	{
		private bool success;

		private long errorCode;

		private string errorDescription;

		protected internal ResponseContent()
			: base()
		{
		}

		public ResponseContent(bool success, long errorCode, string errorDescription)
			: base()
		{
			this.success = success;
			this.errorCode = errorCode;
			this.errorDescription = errorDescription;
		}

		public virtual bool Success
		{
			get
			{
				return success;
			}
			set
			{
				bool success = value;
				this.success = success;
			}
		}

		public virtual long ErrorCode
		{
			get
			{
				return errorCode;
			}
			set
			{
				long errorCode = value;
				this.errorCode = errorCode;
			}
		}

		public virtual string ErrorDescription
		{
			get
			{
				return errorDescription;
			}
			set
			{
				string errorDescription = value;
				this.errorDescription = errorDescription;
			}
		}
	}
}

File: src/test/java/org/codejuicer/java2csharp/testclasses/ResponseContent.java
/*
 * Copyright 2014 Giuseppe Gerla. All Rights Reserved.
 *
 * Licensed under the GNU General Public License, Version 2.0;
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.gnu.org/licenses/gpl2.txt
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package org.codejuicer.java2csharp.testclasses;

public class ResponseContent {
	private boolean success;

	private long errorCode;

	private String errorDescription;

	protected ResponseContent() {
		super();
	}

	public ResponseContent(boolean success, long errorCode,
			String errorDescription) {
		super();
		this.success = success;
		this.errorCode = errorCode;
		this.errorDescription = errorDescription;
	}

	public boolean isSuccess() {
		return success;
	}

	public void setSuccess(boolean success) {
		this.success = success;
	}

	public long getErrorCode() {
		return errorCode;
	}

	public void setErrorCode(long errorCode) {
		this.errorCode = errorCode;
	}

	public String getErrorDescription() {
		return errorDescription;
	}

	public void setErrorDescription(String errorDescription) {
		this.errorDescription = errorDescription;
	}

}